src/sql.rs

```diff
@@ -242,7 +242,7 @@ impl Sql {
         match self.query_get_value_result(query, params) {
             Ok(res) => res,
             Err(err) => {
-                error!(context, "sql: Failed query_row: {}", err);
+                warn!(context, "sql: Failed query_row: {}", err);
```

Comment:
hum, it sounds pretty serious to fail to access the db properly.  Is it really correct to just warn and forget about it?

Reply:
There is a `query_get_value_result` that should be used when errors are handled properly.

The error is not handled anyway, but when desktop is started and has a lot of messages to download, read operations fail because messages are constantly written to the database. And spamming the user with "DC_EVENT_ERROR" messages does not help.